test/test.js

```diff
@@ -23,6 +23,48 @@ function renderFixture(fixtureName, locals) {
 describe('Render', function () {
   jsdom();
 
+  it('should throw a parser error with filename and line number when rendering a broken template', function () {
+    const file = 'attributes-break-parser'
```

Comment:
Oh, I just copied the attributes.jade file, I didn't really think about it.  We can strip the attributes- prefix.